### ISSUE TITLE
Does this worflow support lower body too?

### ISSUE THREAD

**Issue by mr-asleep** [TemryL/ComfyUI-IDM-VTON#77]
Hi, Is this comfyui workflow direct implementation of this model? - https://huggingface.co/spaces/yisol/IDM-VTON

Also does this support lower body too? For jeans and pants.

**Comment by TemryL** [comment]
Yes, it is essentially a wrapper node to run the IDM VTON model (which uses diffusers) from ComfyUI.
It should work with lower body garments as well, provided the mask is generated properly. However, I suspect the results may not be as good for lower body garments, as I imagine the model was primarily trained on upper body setups.

**Comment by fitFusionx** [comment]
I am getting started with this workflow, however the results are a but different than the https://huggingface.co/spaces/yisol/IDM-VTON demo. Am I missing any parameter/node in the attached workflow

<img width="1203" alt="Screenshot 2024-09-03 at 20 31 46" src="https://github.com/user-attachments/assets/aff1a376-d1ad-485e-8fcf-f61099251869">

**Comment by TemryL** [comment]
The mask generated by SAM looks wrong. I guess you want to mask only the upper-body without masking the arms. I would try other prompt than tshirt or different threshold, or you can just generate the mask manually.
